Helpers/MediaLaunchObject.m

```diff
@@ -0,0 +1,46 @@
+//
+//  MediaLaunchObject.m
+//  ConnectSDK
+//
+//  Created by Ibrahim Adnan on 1/19/15.
+//  Copyright (c) 2015 LG Electronics. All rights reserved.
+//
+
+#import "MediaLaunchObject.h"
+
+@implementation MediaLaunchObject
+
+// return nil here so that we force users to have URL and mime-type
+- (instancetype) init
+{
+    return nil;
```

Comment:
Why not throw an exception?
In this `nil` case, it may be unclear why the object isn't created. Especially, with the compiled framework, where the implementation file with the comment is unavailable.